apps/response-api/.env.example

```
@@ -5,3 +5,12 @@ HATCHET_LOG_LEVEL=DEBUG
 
 PORT=7078
 CORS_ALLOWED_ORIGINS=http://localhost:3002,http://127.0.0.1:3002,https://pwa.klicker.com,https://assessment.klicker.com
+
+APP_ORIGIN_AUTH="http://127.0.0.1:3010"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)